pkg/scheduler/objects/allocation.go

```diff
@@ -118,6 +118,10 @@ func NewAllocationFromSI(alloc *si.Allocation) *Allocation {
 	return NewAllocation(alloc.UUID, alloc.NodeID, ask)
 }
 
+func (a *Allocation) GetPartitionNameWithoutClusterID() string {
+	return common.GetPartitionNameWithoutClusterID(a.PartitionName)
```

Comment:
Do we need this method? "common" scope is already available in handlers.go. Can we use it directly in handlers.go?